### ISSUE TITLE
Question about pre-trained classes for pre-trained vgg of ZS-SBIR 

### ISSUE THREAD

**Issue by tiangarin** [gr8joo/IIAE#1]
First of all, thank you for your excellent work.
I read the papers and codes of yours and sem-pcyc's. 
I notice that the classes used for pre-training vgg model in sem-pcyc is not declared, and sem-pcyc randomly choose 25 classes during training (it means that sem-pcyc actually violate strict zero-shot setting).
So I wonder how you choose your train classes and test classes for Sketchy? Just randomly choose?

**Comment by gr8joo** [comment]
Hi, thank you for your kind words.

Although SEM-PCYC did not declare the train / test classes, they have consistently splitted the class by fixing random seed as follows:
np.random.seed(0)
tr_classes = np.random.choice(classes, int(0.8 * len(classes)), replace=False)

Here is the reference [code](https://github.com/AnjanDutta/sem-pcyc/blob/2e083facb54b778861c44f4f32cb28f62d559366/src/utils.py).

When I contacted the authors of SEM-PCYC by email before (because of the question same as yours), they confirmed that the pre-trained VGG-Net is fine-tuned only with the training classes same as the ones used for training their main model.
Using SEM-PCYC's implementation, I extracted exactly the same training / testing splits.

Please let me know if you have further questions.

**Comment by tiangarin** [comment]
Thank you for your detailed reply. It resolves my doubts.
